travel_platform/test_full_landing/readme.txt

```diff
@@ -0,0 +1,2 @@
+This directory (test_full_landing) contains source html and all additional files of landing page of Travel_platform project.
```

Comment:
Well, it's not RST format. See for details: http://draft-edx-style-guide.readthedocs.io/en/latest/ExampleRSTFile.html

Also, it's a common practice to name "README.rst" instead of "readme.rst"